data/ecosystems/a/avalanche.toml

```diff
@@ -597,8 +598,8 @@ url = "https://github.com/ava-labs/glitch-hackathon-test"
 url = "https://github.com/ava-labs/gnosis-subnet"
 
 [[repo]]
-url = "https://github.com/ava-labs/go-ethereum"
 tags = ["Protocol"]
+url = "https://github.com/ava-labs/go-ethereum"
```

Comment:
can you keep the tags parameter below the URL like it was originally? not sure why it was changed